### PR TITLE
updated @nuxtjs/svg @nuxtjs/composition-api 

### PR DESCRIPTION
**Pull request by kb-0311**
## Fixes

Related to #1086 by @krysal 

## Description

updated deps ->
```
"@nuxtjs/svg": "^0.4.0",
"@nuxtjs/composition-api": "^0.32.0",
```

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by kb-0311** [comment]
I have updated some outdated `@nuxt/` dependencies .

Side Notes : 

1. `build-url` dep is deprecated as of now.
2. `vue-i18n` update causes breaking changes.
3. testing libraries and a few other outdated libraries can only be updated once we migrate to nuxt 3 and webpack 5 .

**Review by sarayourfriend — APPROVED** [review]
This will need a rebase for `pnpm-lock.yaml` but otherwise this LGTM! Thanks again for updating these :tada:

**Comment by kb-0311** [comment]
I just rebased on to main and tried to fix the conflicts but something went completely wrong what even happened ?

**Comment by kb-0311** [comment]
I think I figured out what was wrong but I am still not sure please do a review again and let me know if there are some other changes necessary.

**Comment by kb-0311** [comment]
Hmm there are still a few playwright errors for some reason. 

I am not sure whether it was due to the commit?

If yes please suggest some way to resolve it

**Comment by sarayourfriend** [comment]
It looks like the rebase went poorly, there are unrelated code changes in the branch now :disappointed:

**Comment by sarayourfriend** [comment]
@kb-0311 if you can enable edits on the branch by maintainers I can push a fix for the rebase. Fixing the rebase also fixed the playwright tests.

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -99,13 +99,18 @@ dependencies:
...
   '@nuxtjs/redirect-module': 0.3.1
   '@nuxtjs/sentry': 5.1.6
   '@nuxtjs/sitemap': 2.4.0
+<<<<<<< HEAD
   '@nuxtjs/svg': 0.3.0
   '@pinia/nuxt': 0.1.8_pinia@2.0.13
+=======
+  '@nuxtjs/svg': 0.4.0
+  '@pinia/nuxt': 0.1.8_pinia@2.0.11
+>>>>>>> updated @nuxtjs/svg @nuxtjs/composition-api vue-i18n
```
Comment (obulat) [inline pnpm-lock.yaml:113]:
```suggestion
  '@nuxtjs/svg': 0.4.0
  '@pinia/nuxt': 0.1.8_pinia@2.0.13
```

**Review by obulat — APPROVED** [review]
Sorry it took me so long to review this PR, @kb-0311 ! I've rebased it on top of main, and everything works perfectly. Thank you for your contribution!

**Comment by kb-0311** [comment]
thanks a lot @obulat @sarayourfriend . I have ben afk for 3 days now so could not reach out sooner but I am glad this is merged now 👍 .
Thanks for your help !

**Pull request merged**